### PR TITLE
Update Get-CsTeamsAppSetupPolicy.md

### PR DESCRIPTION
**Pull request by xuanzhou4**
Add some usage examples and parameters description for Get-CsTeamsAppSetupPolicy.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: skype/skype-ps/skype/Get-CsTeamsAppSetupPolicy.md [hunk 1]
```diff
@@ -42,8 +42,45 @@ Apps are pinned to the app bar. This is the bar on the side of the Teams desktop
 ## EXAMPLES
 
 ### Example 1
-Intentionally not provided
 
+Get the Global Teams App Setup Policy.
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsTeamsAppSetupPolicy.md:46]:
```suggestion
Get the Global Teams App Setup Policy.

```

Comment (get-itips) [inline skype/skype-ps/skype/Get-CsTeamsAppSetupPolicy.md:46]: please move the description of the example below the code (Leaving a carrier return too)

**Review by dariomws — CHANGES_REQUESTED** [review]

File: skype/skype-ps/skype/Get-CsTeamsAppSetupPolicy.md (hunk 1 above)
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsTeamsAppSetupPolicy.md:46]:
```suggestion
Get the global Teams app setup policy.
```

File: skype/skype-ps/skype/Get-CsTeamsAppSetupPolicy.md [hunk 2]
```diff
@@ -42,8 +42,45 @@ Apps are pinned to the app bar. This is the bar on the side of the Teams desktop
...
+PinnedMessageBarApps : {}
+AppPresetMeetingList : {}
+Description          :
+AllowSideLoading     : True
+AllowUserPinning     : True
+```
+
+### Example 2
+
+Get all the Teams App Setup Policies.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsTeamsAppSetupPolicy.md:62]:
```suggestion
Get all the Teams app setup policies.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @xuanzhou4, kindly commit the suggested changes. Many thanks.

File: skype/skype-ps/skype/Get-CsTeamsAppSetupPolicy.md (hunk 1 above)
Comment (xuanzhou4) [reply skype/skype-ps/skype/Get-CsTeamsAppSetupPolicy.md]: thanks for your reminder, fixed it.

File: skype/skype-ps/skype/Get-CsTeamsAppSetupPolicy.md [hunk 3]
```diff
@@ -42,8 +42,45 @@ Apps are pinned to the app bar. This is the bar on the side of the Teams desktop
...
+ Get-CsTeamsAppSetupPolicy -Identity Global
+
+Identity             : Global
+AppPresetList        : {Id=d2c6f111-ffad-42a0-b65e-ee00425598aa}
+PinnedAppBarApps     : {Id=14d6962d-6eeb-4f48-8890-de55454bb136;Order=1, Id=86fcd49b-61a2-4701-b771-54728cd291fb;Order=2, Id=2a84919f-59d8-4441-a975-2a8c2643b741;Order=3, Id=ef56c0de-36fc-4ef8-b417-3d82ba9d073c;Order=4…}
+PinnedMessageBarApps : {}
+AppPresetMeetingList : {}
+Description          :
+AllowSideLoading     : True
+AllowUserPinning     : True
```
[the remark this answers is not in the document]
Comment (xuanzhou4) [reply skype/skype-ps/skype/Get-CsTeamsAppSetupPolicy.md:58]: fixed it.

File: skype/skype-ps/skype/Get-CsTeamsAppSetupPolicy.md [hunk 4]
```diff
@@ -42,8 +42,45 @@ Apps are pinned to the app bar. This is the bar on the side of the Teams desktop
...
+
+Identity             : Tag:Set-test
+AppPresetList        : {Id=d2c6f111-ffad-42a0-b65e-ee00425598aa}
+PinnedAppBarApps     : {Id=14d6962d-6eeb-4f48-8890-de55454bb136;Order=1, Id=86fcd49b-61a2-4701-b771-54728cd291fb;Order=2, Id=2a84919f-59d8-4441-a975-2a8c2643b741;Order=3, Id=ef56c0de-36fc-4ef8-b417-3d82ba9d073c;Order=4…}
+PinnedMessageBarApps : {}
+AppPresetMeetingList : {}
+Description          :
+AllowSideLoading     : True
+AllowUserPinning     : True
```
[the remark this answers is not in the document]
Comment (xuanzhou4) [reply skype/skype-ps/skype/Get-CsTeamsAppSetupPolicy.md:85]: fixed it.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws @get-itips

**Review by get-itips — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**